docs/guides/references/experiments.mdx

```diff
@@ -25,13 +25,64 @@ configuration to Cypress.
 | `experimentalWebKitSupport`                   | `false` | Enable experimental support for running tests in WebKit. When set, installs of `playwright-webkit` will be detected and available in Cypress. See [Launching Browsers](/guides/guides/launching-browsers#WebKit-Experimental) for more information.                                                                            |
 
+### Experimental CSP Allow List
+
+Cypress will (by default) strip all CSP headers from the response before it is
+sent to the browser. This option allows for more granular control over which CSP
+directives are stripped from the CSP response headers, allowing you to test your
+application with CSP enabled.
+
+#### Strip All CSP Headers
+
+The default value of `experimentalCspAllowList` is `false`. This will remove all
+CSP headers from the response before it is sent to the browser. This option
+should be used if you do not depend on CSP for any tests in your application.
+
+#### Strip Minimum CSP Directives
+
+If you need to test your application with CSP enabled, you can set the
+`experimentalCspAllowList` option to `true`. This will allow all CSP headers to
+be sent to the browser _*except*_ those that could prevent Cypress from
+functioning normally. The following CSP directives are automatically stripped
+from the CSP headers when `experimentalCspAllowList` is set to `true`:
+
+| Stripped Directive           | Allowable | Reason                                                           |
+| ---------------------------- | --------- | ---------------------------------------------------------------- |
+| `default-src`                | Yes       | Conditionally prevents Cypress from loading scripts and running. |
+| `frame-ancestors`            | No        | Prevents Cypress from loading a test application into an iframe. |
+| `require-trusted-types-for`, | No        | Might prevent Cypress from rewriting the DOM.                    |
+| `sandbox`                    | Yes       | Can restrict access to script and iframe functionality.          |
+| `form-action`                | Yes       | Can prevent Cypress from monitoring form events.                 |
+| `navigate-to`                | Yes       | Affects Cypress' ability to navigate to different URLs.          |
+| `script-src`                 | Yes       | Conditionally prevents Cypress from loading scripts and running. |
+| `script-src-elem`            | Yes       | Conditionally prevents Cypress from loading scripts and running. |
+| `trusted-types`              | No        | Could cause Cypress injections to be marked as untrusted.        |
+
+#### Allow Specific CSP Directives
+
+If you need to test your application with CSP enabled, and need to test against
+certain stripped CSP directives, you can set the `experimentalCspAllowList`
+option to an array of directive names. This will allow the specified CSP
+directives to be sent to the browser _so long as_ those directives are maked as
```

Comment:
It looks like the word pulled into this line should probably be "marked" not "maked"!?

Reply:
Fixed.